.github/workflows/build-docker.yaml

```diff
@@ -8,12 +8,21 @@ on:
             - completed
         branches:
             - main
+    push:
+        tags:
+            - '**'
```

Comment:
I'm not sure if this will ever trigger when we use another workflow to tag a commit. Last time I looked workflows could not trigger other workflows.